src/referencePackageSourceGenerator/ReferencePackageSourceTask/GenerateProject.cs

```diff
@@ -135,12 +135,19 @@ public override bool Execute()
                     }
                 }
 
+                propertiesByTfm += $"  <PropertyGroup Condition=\" '$(TargetFramework)' == '{targetFramework}' \">\n";
                 if (subPath == "lib")
                 {
-                    outputPathByTfm += $"  <PropertyGroup Condition=\" '$(TargetFramework)' == '{targetFramework}' \">\n";
-                    outputPathByTfm += $"    <OutputPath>$(ArtifactsBinDir){relativePath}/{subPath}/</OutputPath>\n";
-                    outputPathByTfm += $"  </PropertyGroup>\n\n";
+                    propertiesByTfm += $"    <OutputPath>$(ArtifactsBinDir){relativePath}/{subPath}/</OutputPath>\n";
                 }
+                if (targetFramework == "netstandard2.0" ||
+                    targetFramework == "netstandard2.1" ||
+                    targetFramework == "netcoreapp3.0" ||
+                    targetFramework == "net6.0")
```

Comment:
Are we going to need this for all versions going forward?  If so, can we make this check more abstract (e.g. netx.y)?

Reply:
I want to send a follow-up PR anyway that cleans this up. My idea is that we shouldn't disable implicit framework references at all and just make sure that the right packages are picked up (for all target frameworks). I'm fine with this change and think we should take it as is, as it unblocks targeting net6.0.